storage/campaign.go

```diff
@@ -197,3 +198,13 @@ func NameLike(name string) func(*gorm.DB) *gorm.DB {
 func NotDeleted(db *gorm.DB) *gorm.DB {
 	return db.Where("deleted_at IS NULL")
 }
+
+// GetScheduledCampaigns returns all scheduled campaigns < time
+func (db *store) GetScheduledCampaigns(time time.Time) ([]*entities.CampaignSchedule, error) {
+	var campaignsSchedule []*entities.CampaignSchedule
```

Comment:
No need for a slice of pointers here.
```suggestion
	var campaignsSchedule []entities.CampaignSchedule
```